docs/write-a-plugin.md

```diff
@@ -246,3 +246,41 @@ window.$docsify = {
   ],
 };
 ```
+
+### Theme Swich
+
+It wiil add a theme switch to the after the sidebar and auto add the dark class to the body element, so it you can control the theme by css `body.dark`
```

Comment:
Thanks for the PR.

This should need to be used in conjunction with the theme, so it doesn't fit here.